internal/static/internal/css/frontpage.css

```diff
@@ -119,3 +119,61 @@
     margin-bottom: 0.4em;
     box-sizing: border-box;
 }
+
+/* kode for slideshow */
+@keyframes slidy {
+    0% { left: 50%; }
+    5% { left: 0%; }
+    20% { left: 0%; }
+    25% { left: -100%; }
+    45% { left: -100%; }
+    50% { left: -200%; }
+    70% { left: -200%; }
+    75% { left: -300%; }
+    95% { left: -300%; }
+    100% {left: -400%; }
+    } 
+       
+
+
+
+body { margin: 0; } 
```

Comment:
We already have a root tag dealing with this would just be clutter. See common/static/css/main if you're curious.